gradle.properties

```diff
@@ -97,38 +97,32 @@ kotlin.mpp.import.enableKgpDependencyResolution=true
-# To know which version, should be used, see compose/frameworks/support/libraryversions.toml
-artifactRedirecting.publication=true
-# Look for `COMPOSE` in libraryversions.toml
-artifactRedirecting.androidx.compose.version=1.8.0-beta02
-# Look for `COMPOSE_MATERIAL3` in libraryversions.toml
-artifactRedirecting.androidx.compose.material3.version=1.3.1
-artifactRedirecting.androidx.compose.foundation.version=1.8.0-beta02
-artifactRedirecting.androidx.compose.material.version=1.8.0-beta02
-# Look for `COMPOSE_MATERIAL3_COMMON` in libraryversions.toml
-artifactRedirecting.androidx.compose.material3.common.version=1.0.0-alpha01
-# Look for `COMPOSE_MATERIAL3_ADAPTIVE` in libraryversions.toml
-artifactRedirecting.androidx.compose.material3.adaptive.version=1.1.0-beta01
-# We use artifactRedirecting not only for Compose libs:
-artifactRedirecting.androidx.collection.version=1.5.0
-artifactRedirecting.androidx.annotation.version=1.9.1
-artifactRedirecting.androidx.graphics.version=1.0.1
-artifactRedirecting.androidx.lifecycle.version=2.9.0-alpha12
-artifactRedirecting.androidx.navigation.version=2.9.0-alpha08
-artifactRedirecting.androidx.performance.version=1.0.0-alpha01
-artifactRedirecting.androidx.savedstate.version=1.3.0-alpha10
-artifactRedirecting.androidx.window.version=1.4.0-alpha04
+# When user adds a dependency on `org.jetbrains.compose.*` in Android target,
+# in fact `android.compose` is used.
+#
+# Versions should equal the last merged to the branch
+artifactRedirection.version.androidx.compose=1.8.0-beta02
+artifactRedirection.version.androidx.compose.material3=1.3.1
+artifactRedirection.version.androidx.compose.material3.common=1.0.0-alpha01
+artifactRedirection.version.androidx.compose.material3.adaptive=1.1.0-beta01
+artifactRedirection.version.androidx.collection=1.5.0
+artifactRedirection.version.androidx.annotation=1.9.1
+artifactRedirection.version.androidx.graphics=1.0.1
+artifactRedirection.version.androidx.lifecycle=2.9.0-alpha12
+artifactRedirection.version.androidx.navigation=2.9.0-alpha08
+artifactRedirection.version.androidx.performance=1.0.0-alpha01
+artifactRedirection.version.androidx.savedstate=1.3.0-alpha10
+artifactRedirection.version.androidx.window=1.4.0-alpha04
 
 # For the purpose of substituteForRedirectedPublishedDependencies (look for it in the code)
-artifactRedirecting.modules-for-knative-manifest=\
-  :annotation:annotation=artifactRedirecting.androidx.annotation.version,\
-  :collection:collection=artifactRedirecting.androidx.collection.version,\
-  :graphics:graphics-shapes=artifactRedirecting.androidx.graphics.version,\
-  :lifecycle:lifecycle-common=artifactRedirecting.androidx.lifecycle.version,\
-  :lifecycle:lifecycle-runtime=artifactRedirecting.androidx.lifecycle.version,\
-  :lifecycle:lifecycle-viewmodel=artifactRedirecting.androidx.lifecycle.version,\
-  :performance:performance-annotation=artifactRedirecting.androidx.performance.version
+artifactRedirection.modulesForKNativeManifest=\
```

Comment:
(already wrote it during an introduction of this): It seems that it might be determined without an extra config. It's one more unnecessary source of truth

Reply:
I don't have enough context here to be confident to change it. Created https://youtrack.jetbrains.com/issue/CMP-7847/Refactor-artifactRedirection.modulesForKNativeManifest